### ISSUE TITLE
MacOS support

### ISSUE THREAD

**Issue by arieroos** [eulerto/pgquarrel#89]
**Describe the problem you are trying to solve**

I am trying to build the project on my macbook, but I get this error:
```
make[2]: *** No rule to make target `/usr/local/lib/libpq.so', needed by `pgquarrel'.  Stop.
make[1]: *** [CMakeFiles/pgquarrel.dir/all] Error 2
make: *** [all] Error 2
```

**Describe the solution you'd like**

A way to include the postgres libraries for macos. The libraries would be of type .dylib
I don't know enough about CMake to do this myself.

**Comment by eulerto** [comment]
@arieroos did you follow README instructions? If so, could you provide the output of `cmake` and `make VERBOSE=1` steps?
Have in mind that Postgres should be in your PATH or you should specify `CMAKE_PREFIX_PATH` option.

**Comment by arieroos** [comment]
Yes, I did. I followed the steps for UNIX. The problem is that libpq.so doesn't exist. It wouldn't on MacOS, because of the way MacOS does libraries being different than other UNIX systems. I do have a libpq.dylib

Below the outputs as requested:
```
❯ cmake -DCMAKE_INSTALL_PREFIX=$HOME/pgquarrel -DCMAKE_PREFIX_PATH=/usr/local/Cellar/postgresql/12.2 .
-- The C compiler identification is AppleClang 10.0.1.10010046
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
pg_config: /usr/local/Cellar/postgresql/12.2/bin/pg_config
PostgreSQL FOUND:
LIBS: /usr/local/lib/libpgport.a;/usr/local/lib/libpgcommon.a
PostgreSQL LIBRARIES: /usr/local/lib/libpq.so
PostgreSQL LIBRARY DIRS: /usr/local/lib
PostgreSQL INCLUDE DIRS: /usr/local/include/postgresql/server;/usr/local/include
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   mini

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0
```
And
```
❯ make VERBOSE=1
/usr/local/Cellar/cmake/3.16.5/bin/cmake -S/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 -B/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 --check-build-system CMakeFiles/Makefile.cmake 0
/usr/local/Cellar/cmake/3.16.5/bin/cmake -E cmake_progress_start /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/progress.marks
/Library/Developer/CommandLineTools/usr/bin/make -f CMakeFiles/Makefile2 all
/Library/Developer/CommandLineTools/usr/bin/make -f mini/CMakeFiles/mini.dir/build.make mini/CMakeFiles/mini.dir/depend
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 && /usr/local/Cellar/cmake/3.16.5/bin/cmake -E cmake_depends "Unix Makefiles" /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/CMakeFiles/mini.dir/DependInfo.cmake --color=
Dependee "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/CMakeFiles/mini.dir/DependInfo.cmake" is newer than depender "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/CMakeFiles/mini.dir/depend.internal".
Dependee "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/CMakeFiles/mini.dir/depend.internal".
Scanning dependencies of target mini
/Library/Developer/CommandLineTools/usr/bin/make -f mini/CMakeFiles/mini.dir/build.make mini/CMakeFiles/mini.dir/build
[  2%] Building C object mini/CMakeFiles/mini.dir/mini-file.c.o
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini && /Library/Developer/CommandLineTools/usr/bin/cc -Dmini_EXPORTS  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -fPIC   -o CMakeFiles/mini.dir/mini-file.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/mini-file.c
[  5%] Building C object mini/CMakeFiles/mini.dir/mini-parser.c.o
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini && /Library/Developer/CommandLineTools/usr/bin/cc -Dmini_EXPORTS  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -fPIC   -o CMakeFiles/mini.dir/mini-parser.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/mini-parser.c
[  7%] Building C object mini/CMakeFiles/mini.dir/mini-readline.c.o
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini && /Library/Developer/CommandLineTools/usr/bin/cc -Dmini_EXPORTS  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -fPIC   -o CMakeFiles/mini.dir/mini-readline.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/mini-readline.c
[ 10%] Building C object mini/CMakeFiles/mini.dir/mini-strip.c.o
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini && /Library/Developer/CommandLineTools/usr/bin/cc -Dmini_EXPORTS  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -fPIC   -o CMakeFiles/mini.dir/mini-strip.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/mini-strip.c
[ 13%] Linking C shared library libmini.dylib
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini && /usr/local/Cellar/cmake/3.16.5/bin/cmake -E cmake_link_script CMakeFiles/mini.dir/link.txt --verbose=1
/Library/Developer/CommandLineTools/usr/bin/cc  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -dynamiclib -Wl,-headerpad_max_install_names  -o libmini.dylib -install_name /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini/libmini.dylib CMakeFiles/mini.dir/mini-file.c.o CMakeFiles/mini.dir/mini-parser.c.o CMakeFiles/mini.dir/mini-readline.c.o CMakeFiles/mini.dir/mini-strip.c.o
[ 13%] Built target mini
/Library/Developer/CommandLineTools/usr/bin/make -f CMakeFiles/pgquarrel.dir/build.make CMakeFiles/pgquarrel.dir/depend
cd /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 && /usr/local/Cellar/cmake/3.16.5/bin/cmake -E cmake_depends "Unix Makefiles" /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0 /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/pgquarrel.dir/DependInfo.cmake --color=
Dependee "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/pgquarrel.dir/DependInfo.cmake" is newer than depender "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/pgquarrel.dir/depend.internal".
Dependee "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/CMakeFiles/pgquarrel.dir/depend.internal".
Scanning dependencies of target pgquarrel
/Library/Developer/CommandLineTools/usr/bin/make -f CMakeFiles/pgquarrel.dir/build.make CMakeFiles/pgquarrel.dir/build
[ 15%] Building C object CMakeFiles/pgquarrel.dir/src/am.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/am.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/am.c
[ 18%] Building C object CMakeFiles/pgquarrel.dir/src/aggregate.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/aggregate.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/aggregate.c
[ 21%] Building C object CMakeFiles/pgquarrel.dir/src/cast.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/cast.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/cast.c
[ 23%] Building C object CMakeFiles/pgquarrel.dir/src/collation.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/collation.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/collation.c
[ 26%] Building C object CMakeFiles/pgquarrel.dir/src/common.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/common.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/common.c
[ 28%] Building C object CMakeFiles/pgquarrel.dir/src/conversion.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/conversion.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/conversion.c
[ 31%] Building C object CMakeFiles/pgquarrel.dir/src/domain.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/domain.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/domain.c
[ 34%] Building C object CMakeFiles/pgquarrel.dir/src/eventtrigger.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/eventtrigger.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/eventtrigger.c
[ 36%] Building C object CMakeFiles/pgquarrel.dir/src/extension.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/extension.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/extension.c
[ 39%] Building C object CMakeFiles/pgquarrel.dir/src/fdw.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/fdw.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/fdw.c
[ 42%] Building C object CMakeFiles/pgquarrel.dir/src/function.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/function.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/function.c
[ 44%] Building C object CMakeFiles/pgquarrel.dir/src/index.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/index.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/index.c
[ 47%] Building C object CMakeFiles/pgquarrel.dir/src/language.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/language.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/language.c
[ 50%] Building C object CMakeFiles/pgquarrel.dir/src/matview.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/matview.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/matview.c
[ 52%] Building C object CMakeFiles/pgquarrel.dir/src/operator.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/operator.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/operator.c
[ 55%] Building C object CMakeFiles/pgquarrel.dir/src/policy.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/policy.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/policy.c
[ 57%] Building C object CMakeFiles/pgquarrel.dir/src/publication.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/publication.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/publication.c
[ 60%] Building C object CMakeFiles/pgquarrel.dir/src/privileges.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/privileges.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/privileges.c
[ 63%] Building C object CMakeFiles/pgquarrel.dir/src/quarrel.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/quarrel.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/quarrel.c
[ 65%] Building C object CMakeFiles/pgquarrel.dir/src/rule.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/rule.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/rule.c
[ 68%] Building C object CMakeFiles/pgquarrel.dir/src/schema.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/schema.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/schema.c
[ 71%] Building C object CMakeFiles/pgquarrel.dir/src/sequence.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/sequence.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/sequence.c
[ 73%] Building C object CMakeFiles/pgquarrel.dir/src/server.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/server.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/server.c
[ 76%] Building C object CMakeFiles/pgquarrel.dir/src/statistics.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/statistics.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/statistics.c
[ 78%] Building C object CMakeFiles/pgquarrel.dir/src/subscription.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/subscription.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/subscription.c
[ 81%] Building C object CMakeFiles/pgquarrel.dir/src/table.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/table.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/table.c
[ 84%] Building C object CMakeFiles/pgquarrel.dir/src/textsearch.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/textsearch.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/textsearch.c
[ 86%] Building C object CMakeFiles/pgquarrel.dir/src/transform.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/transform.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/transform.c
[ 89%] Building C object CMakeFiles/pgquarrel.dir/src/trigger.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/trigger.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/trigger.c
[ 92%] Building C object CMakeFiles/pgquarrel.dir/src/type.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/type.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/type.c
[ 94%] Building C object CMakeFiles/pgquarrel.dir/src/usermapping.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/usermapping.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/usermapping.c
[ 97%] Building C object CMakeFiles/pgquarrel.dir/src/view.c.o
/Library/Developer/CommandLineTools/usr/bin/cc  -I/usr/local/include/postgresql/server -I/usr/local/include -I/Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/mini  -O0 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -fno-strict-aliasing -fwrapv -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -o CMakeFiles/pgquarrel.dir/src/view.c.o   -c /Users/arieroos/Workspace/pgquarrel-pgquarrel_0_7_0/src/view.c
make[2]: *** No rule to make target `/usr/local/lib/libpq.so', needed by `pgquarrel'.  Stop.
make[1]: *** [CMakeFiles/pgquarrel.dir/all] Error 2
make: *** [all] Error 2
```
Here is also the libpq contents of my `/usr/local/lib/`. It might also be useful:
```
❯ ll | grep libpq
lrwxr-xr-x   1 arieroos  admin    46B Feb 26 11:21 libpq.5.12.dylib -> ../Cellar/postgresql/12.2/lib/libpq.5.12.dylib
lrwxr-xr-x   1 arieroos  admin    43B Feb 26 11:21 libpq.5.dylib -> ../Cellar/postgresql/12.2/lib/libpq.5.dylib
lrwxr-xr-x   1 arieroos  admin    37B Feb 26 11:21 libpq.a -> ../Cellar/postgresql/12.2/lib/libpq.a
lrwxr-xr-x   1 arieroos  admin    41B Feb 26 11:21 libpq.dylib -> ../Cellar/postgresql/12.2/lib/libpq.dylib
```

**Comment by eulerto** [comment]
@arieroos I've committed a fix (cf 37238e57f5ff6720683797db088376f1d62c4ac6). I didn't test it because I don't have a MacOS. Could you try again?

**Comment by arieroos** [comment]
It is working perfectly. Thank you so much!
